partcad/src/partcad/actions/part_actions.py

```
@@ -71,7 +71,7 @@ def import_part_action(project: Project, kind: str, name: str, source_path: str,
         try:
             shutil.copy2(source_path, target_path)
         except shutil.Error as e:
-            raise ValueError(f"Failed to copy '{source_path}' → '{target_path}': {e}")
+            raise ValueError(f"Failed to copy '{source_path}' -> '{target_path}': {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)